docs/admin/cluster-management.md

```diff
@@ -14,47 +14,141 @@ certainly want the docs that go with that version.</h1>
+Addresses:  10.240.109.233,130.211.135.246
+Capacity:
+ cpu:   1
+ memory:  3800808Ki
+ pods:    100
+Version:
+ Kernel Version:    3.16.0-0.bpo.4-amd64
+ OS Image:      Debian GNU/Linux 7 (wheezy)
+ Container Runtime Version: docker://Unknown
+ Kubelet Version:   v0.21.1-151-g2766d3b624b58d-dirty
+ Kube-Proxy Version:    v0.21.1-151-g2766d3b624b58d-dirty
+PodCIDR:      10.244.0.0/24
+ExternalID:     1414038391464598590
+Pods:       (3 in total)
+  Namespace     Name
+  kube-system     fluentd-cloud-logging-kubernetes-minion-0fek
+  kube-system     monitoring-heapster-v5-kyijl
+  kube-system     monitoring-influx-grafana-v1-yqbuh
+Events:
+  FirstSeen       LastSeen      Count From          SubobjectPath Reason    Message
+  Thu, 09 Jul 2015 13:26:10 +0200 Thu, 09 Jul 2015 13:26:10 +0200 1 {kubelet kubernetes-minion-0fek}      NodeNotReady  Node kubernetes-minion-0fek status is now: NodeNotReady
+  Thu, 09 Jul 2015 13:26:12 +0200 Thu, 09 Jul 2015 13:26:12 +0200 1 {kubelet kubernetes-minion-0fek}      NodeNotReady  Node kubernetes-minion-0fek status is now: NodeNotReady
+  Thu, 09 Jul 2015 13:26:42 +0200 Thu, 09 Jul 2015 13:26:42 +0200 1 {kubelet kubernetes-minion-0fek}      starting  Starting kubelet.
+  Thu, 09 Jul 2015 13:26:43 +0200 Thu, 09 Jul 2015 13:26:43 +0200 1 {kubelet kubernetes-minion-0fek}      NodeReady Node kubernetes-minion-0fek status is now: NodeReady
+```
 
+Another thing you can get is a list of Events related to specific component. Events are persisted in etcd and provide some high level information on what was happening in the cluster. To list events you can use
+```
+cluster/kubectl.sh get events
 ```
-$ hack/build-go.sh cmd/kube-version-change
-$ _output/local/go/bin/kube-version-change -i myPod.v1beta3.yaml -o myPod.v1.yaml
+but you have to remember that events are namespaced. This means that if you're interested in events for some namespaced object (e.g. what happened with pods in namespace `my-namespace`) you need to explicitly provide a namespace to this command:
 ```
+cluster/kubectl.sh get events --namespace=my-namespace
+```
+
+Other available introspection/monitoring tools include Elasticsearch, Graphana etc. They are shipped with Kubernetes to provide additional functionality, such as monitoring resource usage, or allowing searching logs. They run as ordinary pods and can be managed in exactly the same way. More detailed description of cluster addons is available [here](../../cluster/addons/).
+
+## Tearing down the cluster
+If you're running your cluster on GKE you can delete it from the web UI, or using `gcloud` CLI. 
+
+Otherwise you tear down the cluster by using `cluster/kube-down.sh` script. On GCE `kube-down.sh` will delete all resources, but depending on the environment you're running in, you may need to clean up some things yourself, such as some network leftovers, persistent storage etc.
 
 ### Maintenance on a Node
 
+TODO: this is out of date, need a rewrite as well.
```

Comment:
I would drop the changes to this section (including the TODO) except for:
- "assuming that not all nodes will go down in the same time" is fine to keep, but change it to "at the same time"
- capitalizing Additionally

pod-eviction-timeout is mentioned in node.md (or it should be mentioned there), it is too much detail to put here.